### ISSUE TITLE
Selection tool -> regex for itemname/lore

### ISSUE THREAD

**Issue by bobhenl** [Zrips/Residence#654]
Hi, could you please add way to specify special regex for itemname/lore to which the creation item won't apply? For ex. if I'll set wooden hoe, then there are 2 items in Slimefun that then stop works... So if I could add them on the allowed lores/names list by this?

Something similar that is in trademe, but probably named allowed-names & lores:
(from Trademe config)
```
BlockedLore:
- One line of lore
- second line
- color code not necessary
```

and may ideally with support for regex

**Comment by Zrips** [comment]
You could simply use different selection tool to avoid any confusion in general, as having tool with specific lore which in general should be obtainable by regular players isint best solution.

**Comment by bobhenl** [comment]
Yeah, but it's the most used on all servers :/ We had bone before, but we 've changed back to the wooden hoe. For this reason will be good to have something like blockedlore or way to add more tools. When we used bone people were asking what to use to create residence, they know mainly about wooden hoe, but then there are problems like this that something like Slimefun use wooden hoe too etc
